### PR TITLE
Changed urls to download from this repo and use the new terraform version

### PR DESCRIPTION
**Pull request by neikei**
Signed-off-by: Niklas Vosskoetter <niklas@vosskoetter.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,11 +40,11 @@ cd ocp4-upi-powervm/
 
 Download the latest Terraform binary for IBM Power from https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/latest/.
 ```
-curl -L https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/latest/terraform-0.13.5 -o /usr/bin/terraform && chmod +x /usr/bin/terraform
+curl -L https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/latest/terraform-0.14.5 -o /usr/bin/terraform && chmod +x /usr/bin/terraform
```
Comment (jarooony) [inline README.md]: raise terraform latest version to 0.14.6
```suggestion
curl -L https://oplab9.parqtec.unicamp.br/pub/ppc64el/terraform/latest/terraform-0.14.6 -o /usr/bin/terraform && chmod +x /usr/bin/terraform
```

Comment (neikei) [reply README.md]: Thanks for the hint. Version 0.14.7 was released today.

**Comment by yussufsh** [comment]
The intention of this project is to support the projects given in https://github.com/ocp-power-automation/terraform-providers-power#terraform-automation to run on Power systems. The given projects are still locked at Terraform v0.13.

We can have this change once the TF modules support Terraform v0.14.

**Comment by neikei** [comment]
That's okay, but it doesn't fit with the advice in the readme to download the latest terraform. We will be installing some OpenShift over the next few weeks and testing the latest version as well as giving feedback to this project.

**Review by yussufsh — APPROVED** [review]
lgtm.

For now we will go with this version. In future the statements will need to be generic. Perhaps we should not focus on "how to install TF on Power" in this repo as this is created for hosting the required providers.

**Pull request merged**